test/integration/failover_stateful_test.lua

```diff
@@ -295,6 +379,9 @@ function g.test_leaderless()
     -- Check cluster repairs
 
     g.kingdom:start()
+    helpers.retrying({}, function()
+        g.kingdom:connect_net_box()
```

Comment:
Is it necessary?

Reply:
To have an ability write test below and return back broken kingdom connection